### PR TITLE
WIP Adding Performance workload for Azure

### PR DESCRIPTION
**Pull request by jtaleric**
Adding a simple FIO workload to look at Azure disk performance

### CODE REVIEW

File: ci-operator/config/cloud-bulldozer/plow/cloud-bulldozer-plow-master.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+base_images:
+  base:
```
Comment (stevekuznetsov) [inline ci-operator/config/cloud-bulldozer/plow/cloud-bulldozer-plow-master.yaml]: Oops, this should have been `cli:`

File: ci-operator/config/cloud-bulldozer/plow/cloud-bulldozer-plow-master.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    tag: cli
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.12
+images:
+- dockerfile_path: azure/Dockerfile
+  from: base
```
Comment (stevekuznetsov) [inline ci-operator/config/cloud-bulldozer/plow/cloud-bulldozer-plow-master.yaml]: And this should have been `src`, sorry

File: ci-operator/config/cloud-bulldozer/plow/cloud-bulldozer-plow-master.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.12
+images:
+- dockerfile_path: azure/Dockerfile
```
Comment (stevekuznetsov) [inline ci-operator/config/cloud-bulldozer/plow/cloud-bulldozer-plow-master.yaml]: Could you try adding under this line

```yaml
  context_dir: azure/
```

**Comment by stevekuznetsov** [comment]
| /test pj-rehearse
-- | --

File: ci-operator/jobs/cloud-bulldozer/plow/cloud-bulldozer-plow-master-presubmits.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
+presubmits:
```
Comment (deads2k) [inline ci-operator/jobs/cloud-bulldozer/plow/cloud-bulldozer-plow-master-presubmits.yaml]: I don't read these yaml's well. A way to optionally run the job makes sense.  Running on every PR is premature.

**Pull request closed without merging**